lib/std/priority_dequeue.zig

```diff
@@ -0,0 +1,879 @@
+const std = @import("std");
```

Comment:
Small nitpick: All files in the standard library have a license identifier at the top. It would make sense to add this here as well.

Reply:
Ah yes, I meant to add this. Thanks for pointing it out, will fix.